### PR TITLE
Add docs for 'Execution'

### PR DESCRIPTION
**Pull request by sreyasaju**

### CODE REVIEW

File: docs/mdbook/src/execution.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Execution
+
+Running sugaroid is easy as pie
+
+Just execute
+
+```bash
+>> sugaroid
```
Comment (srevinsaju) [inline docs/mdbook/src/execution.md:8]: no, I mean, for bash its `$` and for power shell its `>>`

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: docs/mdbook/src/execution.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* update : Running sugaroid update will clear the current database and train the new data and store it persistently to the configuration path as sugaroid.db . (See Configuration for more details)
+
+To launch the sugaroid web server on any IP address, do a local clone of the package by
+
+```bash
+git clone https://github.com/srevinsaju/sugaroid-wsgi --depth=1
+cd sugaroid-wsgi
+python manage.py runserver
+```
+Follow the on-screen instructions to get it running on your web browser. If the command completed with a status OK, you should be able to see sugaroid running on https://0.0.0.0:8000
```
Comment (srevinsaju) [inline docs/mdbook/src/execution.md:28]: Please change `https` to `http`

File: docs/mdbook/src/execution.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+from the Terminal (Linux, Mac OS) and PowerShell (on Windows)
+
+There are few arguments that can be passed to sugaroid
+
+* qt : Running sugaroid qt will start the sugaroid graphical user interface
+* audio : Running sugaroid audio will include audio support for sugaroid (Data charges may apply)
+* train: Running sugaroid train will start the sugaroid trainer, which you can use to train sugaroid for some responses
+* update : Running sugaroid update will clear the current database and train the new data and store it persistently to the configuration path as sugaroid.db . (See Configuration for more details)
```
Comment (srevinsaju) [inline docs/mdbook/src/execution.md:19]: It would be better to surround qt, etc.. with backticks, `qt`, `audio`, `train`, etc..

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: docs/mdbook/src/execution.md (hunk 3 above)
Comment (sreyasaju) [reply docs/mdbook/src/execution.md:19]: Okay :)

File: docs/mdbook/src/execution.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+```bash
+$ sugaroid
+```
+
+
+from the Terminal (Linux, Mac OS) and PowerShell (on Windows)
+
+There are few arguments that can be passed to sugaroid
+
+```qt``` : Running sugaroid qt will start the sugaroid graphical user interface
```
Comment (srevinsaju) [inline docs/mdbook/src/execution.md:16]: Not triple backticks, its single

Comment (sreyasaju) [reply docs/mdbook/src/execution.md]: :sweat_smile:

**Review by srevinsaju — APPROVED** [review]

**Pull request merged**